lib/puppet/indirector/rest.rb

```diff
@@ -100,6 +97,7 @@ def self.port
           Puppet.settings[setting].to_i
         end
       end
+      port
```

Comment:
We no longer call `to_i` in the case `serverport` is found in the context (which I think explains the change to `rest_spec`). Maybe change this line to `port.to_i` and remove the `to_i` calls in the fallback block.